tests/foreman/ui/test_errata.py

```diff
@@ -60,7 +60,7 @@
 RHVA_ERRATA_ID = REAL_4_ERRATA_ID
 RHVA_ERRATA_CVES = REAL_4_ERRATA_CVES
 
-pytestmark = ['run_in_one_thread']
+pytestmark = [pytest.mark.run_in_one_thread]
```

Comment:
Not sure why we are saving the mark in a list here ? Cant we directly use it ?

(Cannot see its usage since I am reviewing from my Mobile with Github App)

Reply:
Also Make sure you have imported pytest in all these test modules for this change if not already.

Reply:
There is `import pytest` everywhere otherwise it would fail.

Scope of this PR is to fix urgent test collection error. Please don't ask me such questions about why it's there.
(you can investigate by git blame for yourself history of this code)

Reply:
Since I was doing the review from Mobile, The App didnt show me pytest is imported as it doesnt shows the full file while reviewing.

Anyways, I got to know the purpose behind `pytestmark` as it helps to globally apply mark for all the tests in the module.